crates/holochain/src/conductor/conductor.rs

```diff
@@ -1261,7 +1261,7 @@ mod app_impls {
                     let stopped_app = self.add_disabled_app_to_db(app).await?;
 
                     // Return the result, which be may be an error if no_rollback was specified
-                    genesis_result.map(|()| stopped_app.into())
+                    genesis_result.map(|()| stopped_app)
```

Comment:
I think there are no "stopped apps" anymore, right? So we might rename these occurrences of `stopped_app` with `disabled_app`? There are also some usages of `stopped_app` in `crates/holochain/src/conductor/state.rs`.

Reply:
Yes, it's good to replace that throughout.

Reply:
There are more occurrences of `stopped_app` in `crates/holochain/src/conductor/state.rs` as I wrote above. It's not strictly necessary for this PR but would be nice to replace as well I think.